apps/omg_watcher/lib/block_getter.ex

```diff
@@ -40,18 +41,60 @@ defmodule OMG.Watcher.BlockGetter do
     GenServer.call(__MODULE__, :get_events)
   end
 
-  @spec download_block(pos_integer()) :: Core.validate_download_response_result_t()
-  defp download_block(requested_number) do
-    {:ok, {requested_hash, block_timestamp}} = Eth.RootChain.get_child_chain(requested_number)
-    response = Client.get_block(requested_hash)
+  def start_link(_args) do
```

Comment:
I only moved functions around. API at the top, callbacks and finally private functions.